src/components/validations/with-validation.hoc.js

```diff
@@ -272,8 +272,8 @@ const withValidation = (WrappedComponent) => {
     info: []
   };
 
-  const displayName = WrappedComponent.displayName || WrappedComponent.name || 'Component';
-  WithValidation.displayName = `WithValidation(${displayName})`;
+  const displayName = WrappedComponent.displayName || WrappedComponent.name || 'WithValidation(Unknown)';
+  WithValidation.displayName = displayName.replace('Base', '');
```

Comment:
I guess the substring `'Base'` will always occur at the beginning of `displayName`? For example:

* `BaseButtonToggleGroup`
* `BaseCarousel`
* `BaseDateInput`
* `BaseForm`

If so, maybe using a regex with a start-of-string anchor `^` would be a bit more robust and also make its behaviour clearer for future developers?

```suggestion
  WithValidation.displayName = displayName.replace(/^Base/, '');
```

Reply:
Fixed in 2f027d8

Reply:
Thanks :thumbsup: